samples/bluetooth/central_hr/Kconfig.sysbuild

```diff
@@ -13,3 +13,28 @@ config NET_CORE_IMAGE_HCI_IPC
 	bool "HCI IPC image on network core"
 	default y
 	depends on NET_CORE_BOARD != ""
+
+config BT_CTLR_SCAN_DATA_LEN_MAX
+	int "Scan data length max"
+	default 36
+	depends on TEST_MODE_EXTENDED || TEST_MODE_PHY_CODED
```

Comment:
given that the sysbuild code if's on these, I guess this depends is not really needed?
(same for BT_BUF_EVT_DISCARDABLE_SIZE)